crates/local-cluster-runner/src/node/mod.rs

```diff
@@ -558,6 +565,23 @@ impl StartedNode {
         &self.config().common.advertised_address
     }
 
+    pub async fn last_n_lines(&self, n: usize) -> Result<Vec<String>, rev_lines::RevLinesError> {
+        let log_file = self.log_file.clone();
+        match tokio::task::spawn_blocking(move || {
+            let log_file = std::fs::File::open(log_file)?;
+            let mut lines = Vec::with_capacity(n);
+            for line in RevLines::new(log_file).take(n) {
+                lines.push(line?)
+            }
+            Ok(lines)
+        })
+        .await
+        {
+            Ok(res) => res,
+            Err(_) => Err(io::Error::other("background task failed").into()),
+        }
```

Comment:
Can't push the fix to your fork, this will fix the Clippy warning:

```suggestion
        tokio::task::spawn_blocking(move || {
            let log_file = std::fs::File::open(log_file)?;
            let mut lines = Vec::with_capacity(n);
            for line in RevLines::new(log_file).take(n) {
                lines.push(line?)
            }
            Ok(lines)
        })
        .await
        .unwrap_or_else(|_| Err(io::Error::other("background task failed").into()))
```